### PR TITLE
Extend Python documentation

### PR DESCRIPTION
**Pull request by matiasinsaurralde**
Add instructions for RedHat/CentOS. Clarify Python versioning and requirements.

### CODE REVIEW

**Review by marksou — APPROVED** [review]
LGTM

File: tyk-docs/content/plugins/rich-plugins/python/python.md [hunk 1]
```diff
@@ -15,18 +15,50 @@ Since v2.9, Tyk supports any currently stable [Python 3.x version](https://www.p
...
+If you're using Ubuntu/Debian:
 
-Install the build tools: `apt-get install -y build-essential`
+```{.copyWrapper}
+apt install python3 python3-dev python3-pip build-essential
+```
 
-### Install the Required Modules
+If you're using RedHat or CentOS, Yum could be used.
+Note that this sample refers to Python 3.4, you should use a suffix that matches the Python version you're planning to use:
```
Comment (excieve) [inline tyk-docs/content/plugins/rich-plugins/python/python.md]: I'd suggest not mentioning Python 3.4 at all — it's EOL.

File: tyk-docs/content/plugins/rich-plugins/python/python.md [hunk 2]
```diff
@@ -15,18 +15,50 @@ Since v2.9, Tyk supports any currently stable [Python 3.x version](https://www.p
...
+
+```{.copyWrapper}
+yum install python34-dev python34-setuptools
+easy_install-3.4 pip
```
Comment (excieve) [inline tyk-docs/content/plugins/rich-plugins/python/python.md]: Better yet `python3 -m ensurepip` if `pip` is for some reason not in the package repo.

File: tyk-docs/content/plugins/rich-plugins/python/python.md (hunk 1 above)
Comment (matiasinsaurralde) [reply tyk-docs/content/plugins/rich-plugins/python/python.md]: Makes sense, do you know which latest version is available in the new RHEL/CentOS setups?

**Comment by buger** [comment]
@matiasinsaurralde pls address comments and lets merge it.

**Pull request merged**